src/Psd/PsdInterleave.cpp

```diff
@@ -5,11 +5,25 @@
 #include "PsdInterleave.h"
 
 #include "PsdUnionCast.h"
+
+#if !defined(USE_SSE)
+#if defined(_M_IX86) || defined(_M_X64)
+#define USE_SSE 1
+#else
+#define USE_SSE 0
+#endif
+#endif
+
+#if USE_SSE
 #include <emmintrin.h>
+#else
+#include <tuple>

```

Comment:
What do you need tuple for?

Reply:
It is required to use **std::ignore**. Otherwise there are warnings for unused parameter **blockSize**.

Reply:
Ah, I see.
There's no need to use std::ignore in that case, however.
Please remove the #include and use `unsigned int /* blockSize */` instead, which achieves the same.

I posted a few options that don't require std::ignore in case you're interested:
https://twitter.com/molecularmusing/status/1481612964771868677

Reply:
Done. Thanks for advice!